### PR TITLE
New Isotopes dialog

### PR DESCRIPTION
**Pull request by shubhra-agrawal**
The "Report Isotopes" button was available in the Peaks dialog and other settings for labeled data lay in the Options dialog. The UI was not intuitive when it came to label detection. Especially for manual peak picking.

All label detection settings have been moved to a new Isotope dialog in the main toolbar for easy access.

Issue: #185

### CODE REVIEW

File: src/gui/mzroll/peakdetectiondialog.cpp [hunk 1]
```diff
@@ -227,29 +208,25 @@ void PeakDetectionDialog::closeEvent(QCloseEvent* event)
...
         mainwindow->alignmentDialog->peakDetectionAlgo->setCurrentIndex(1);
         featureOptions->setChecked(true);
-        reportIsotopesOptions->setEnabled(false);
     }
     
     QString dbName = compoundDatabase->currentText();
     toggleFragmentation(dbName);
 }
 
+bool PeakDetectionDialog::getDbOptions()
```
Comment (saifulbkhan) [inline src/gui/mzroll/peakdetectiondialog.cpp]: You don't have to make this change but the name of this method can be a bit better (suggestion: `isDatabaseSearchSelected`).

Comment (saifulbkhan) [reply src/gui/mzroll/peakdetectiondialog.cpp]: or maybe `isDatabaseSearchEnabled`?

File: src/gui/mzroll/isotopedialog.cpp [hunk 2]
```diff
@@ -76,3 +88,8 @@ void IsotopeDialog::onReset()
 {
     emit resetSettings(isotopeSettings->getSettings().keys());
 }
+
+bool IsotopeDialog::getReportIsotopes()
```
Comment (saifulbkhan) [inline src/gui/mzroll/isotopedialog.cpp]: This can also have a better name, such as - `isIsotopeReportingEnabled`.

**Comment by shubhra-agrawal** [comment]
@saifulbkhan Made the required changes. Please take a look.

> remaining 4 isotope flags change from C13Labeled_BPE, ... etc., which was probably named like that because it was meant for the bar-plot widget
The barplot flags still exist, as the checkboxes are part of the barplot widget. The isotopeWidget version has been removed and merged with `C13Labeled_BPE` and left as is to avoid confusion with `C13Labeled_Barplot`

**Comment by saifulbkhan** [comment]
@shubhra-agrawal Thanks. Changes look good. Go ahead and merge if you feel all the necessary work is done.

**Pull request merged**